### ISSUE TITLE
fails to drop_chunks on a hyper  table

### ISSUE THREAD

**Issue by zivyankelevitz** [timescale/timescaledb#2444]
**Relevant system information:**
 - OS: [Windows 10 x64]
 - PostgreSQL version (output of `postgres --version`): [10]
 - TimescaleDB version (output of `\dx` in `psql`): [1.7.3]
 - Installation method: ["using Docker"]

**Describe the bug**
i'm trying to drop chunks from a hypertable under some schema(not public). the hyper table has a continuous_aggregates view:
SELECT drop_chunks(interval '2 days', 'tpadataaccess.tpa_tie')

i get the following response:
ERROR: "tpadataaccess.tpa_tie" is not a hypertable or a continuous aggregate view
SQL state: TS001
Hint: It is only possible to drop chunks from a hypertable or continuous aggregate view

but this is a hyper table.
i'm validating it by running this command:
SELECT create_hypertable('tpadataaccess.tpa_tie', 'time', chunk_time_interval => interval '1 day');

and i get the following response:
ERROR: table "tpa_tie" is already a hypertable
SQL state: TS110

**Expected behavior**
chunks are deleted from hyper table

**Actual behavior**
ERROR: "tpadataaccess.tpa_tie" is not a hypertable or a continuous aggregate view
SQL state: TS001
Hint: It is only possible to drop chunks from a hypertable or continuous aggregate view

**Screenshots**
![image](https://user-images.githubusercontent.com/70508840/94175320-31624380-fe9f-11ea-8618-bf230f8fceed.png)

**Comment by kmatt** [comment]
- CentOS Linux release 8.2.2004 (Core)
- Postgresql 12.3
- timescaledb  1.7.2


```
# select show_chunks('prometheus.metrics');
                show_chunks
--------------------------------------------
 _timescaledb_internal._hyper_14_1976_chunk
 _timescaledb_internal._hyper_14_1977_chunk
...

# SELECT drop_chunks(INTERVAL '1 day', table_name => 'prometheus.metrics');
ERROR:  "prometheus.metrics" is not a hypertable or a continuous aggregate view
HINT:  It is only possible to drop chunks from a hypertable or continuous aggregate view
```

**Comment by mfreed** [comment]
The API for `drop_chunks` currently separates the `schema_name` into a separate argument, rather than allowing you to specify both schema and table name as you try.  (To be honest, I forget why.)

https://docs.timescale.com/latest/api#drop_chunks

Try:

```
SELECT drop_chunks(interval '2 days', table_name => 'tpa_tie', schema_name => 'tpadataaccess');
```

**Comment by mkindahl** [comment]
In TimescaleDB 1.7 the syntax is to separate the table name and the schema name, but in TimescaleDB 2.0 we have changed to use `REGCLASS` for the first parameter instead. See #1905 

Closing since it's not a good idea to APIs in a minor version and this is fixed in 2.0.

**Comment by zivyankelevitz** [comment]
Hi Mats,
it works like a charm now :-)
thanks for the super fast response
this is yet another reason to use this database
all the best and thank you
